litrpc/lit-accounts.proto

```diff
@@ -25,6 +25,18 @@ service Accounts {
     */
     rpc UpdateAccount (UpdateAccountRequest) returns (Account);
 
+    /* litcli: `accounts creditaccount`
```

Comment:
think the call here is wrong? looks like it is `litcli accounts credit`

same for debit